libraries/AP_HAL_ChibiOS/hwdef/KakuteH7Mini-Nand/defaults.parm

```diff
@@ -0,0 +1,7 @@
+# setup for LEDs on chan9
+SERVO9_FUNCTION 120
+NTF_LED_TYPES 257
+# Default VTX power to on
+RELAY_DEFAULT 1
+# UART1 for DJI Goggles
+SERIAL1_PROTOCOL 33
```

Comment:
Good catch. Can you add the DJI protocol to the regular H7 Mini?

Reply:
done